labs/hlf-operator.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Hyperledger Fabric Operator for Kubernetes
+parent: Labs
+---
+# Lab Name
+Hyperledger Fabric Operator for Kubernetes
+
+# Short Description
+Manage Hyperledger Fabric components at scale with a Kubernetes Operator.
+
+# Scope of Lab
+I think there's a gap to deploy a Hyperledger Fabric network in Kubernetes, this project automates to automate the deployment.
+
+The work will be focused on automating the creation of peers, orderers and CAs.
+
+I have some ideas I would like to share such as developing a User Interface, and I think this could be integrated natively with the [fabric-operations-console](https://github.com/hyperledger-labs/fabric-operations-console) which was recently released.
```

Comment:
[nitpick] This statement can be rephrased as follows

```suggestion
The proposal is for the deployment of a Hyperledger Fabric network on Kubernetes. This proposal can be integrated natively with the [fabric-operations-console](https://github.com/hyperledger-labs/fabric-operations-console) which was recently released.
```

Reply:
Added